### DISCUSSION TITLE
Please add a class to "single-event-container.daywrap" for calendar entity

### DISCUSSION THREAD

**Discussion by 3rdinger** [totaldebug/atomic-calendar-revive#1432] [Feature Requests] [answered]
In order to use multiple entities in one calendar, it would be nice to have a different color for each one. E.g. PersonA: green, PersonB blue. So maybe it is possible to add the source entity as a class to the single-event-container.daywrap class. Then it would be possible to add css with card_mod.

**Comment by 3rdinger** [comment]
![p](https://github.com/totaldebug/atomic-calendar-revive/assets/11486602/65822914-673d-45da-99bd-28f5af298179)

**Reply by marksie1988** [reply]
You can set the option "color" under entities to set a color for each calendar already. 

Or do you mean a different colour for different calendar entries in the same calendar?

**Reply by 3rdinger** [reply]
I mean a different color for different calendar entries in the same calendar.
In entity.rst there is a variable called color that seems to make what I want but it doesn't work. Maybe because of "In version 9 of the Atomic Calendar Revive I removed all of the color and style Options" like described in styling.rst.
So if there is a html class of the entity name like "justin" it should be possible to css that.

**Reply by marksie1988** [reply]
The color options is the only one that does still exist. 

Please share your config so I can see why it's not working for you.

**Reply by 3rdinger** [reply, marked as answer]
Sorry... it was my mistake! I did not write a '#' in front of the RGB Code. I gave up trying because of the sentence 'I removed all of the color and style Options'. Now it works! Thank you!

code example if anybody else struggles with that: color: '#ff0000'
